packages/kbn-eslint-plugin-telemetry/helpers/get_app_name.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { camelCase } from 'lodash';
+import path from 'path';
+
+const XPACK_DIR_ROOT = '/x-pack/plugins';
```

Comment:
packages and package-plugins can now be defined all over the repository and it is not guaranteed they will only live on those 2 places. A list of all the folders where there is a package or a package-plugin could be obtained with something like: 

```
Array.from(require('@kbn/repo-packages').getPkgDirMap(REPO_ROOT).values()).reduce((a,c) => { const topD = c.normalizedRepoRelativeDir.split('/')[0]; if (topD) { a.add(topD) } return a; }, new Set())
```